linera-rpc/tests/staged/formats.yaml

```diff
@@ -844,6 +852,14 @@ SystemOperation:
       Admin:
         NEWTYPE:
           TYPENAME: AdminOperation
+TimeoutConfig:
+  STRUCT:
+    - fast_round_duration:
+        TYPENAME: Duration
```

Comment:
Looks like we do serialize durations after all.

Reply:
Right, the `TimeoutConfig` is now part of `ChainOwnership`, which is part of the `ChainManager` as well as some cross-chain messages.

Yes, we should decide at some point whether we want to switch back to microseconds here, or specify a serialization format for `TimeoutConfig`.